app/services/order_cycle_distributed_products.rb

```diff
@@ -13,6 +13,21 @@ def products_relation
     Spree::Product.where(id: stocked_products).group("spree_products.id")
   end
 
+  # Joins on the first product variant to allow us to filter product by taxon. This is so
+  # enterprise can display product sorted by category in a customer order on their shopfront.
```

Comment:
I had to read it a couple of times to work out what it meant ;)
```suggestion
  # enterprise can display product sorted by category in a custom order on their shopfront.
```